lib/src/cli.dart

```diff
@@ -18,41 +19,55 @@ const String APP_VERSION = '0.7.0';
 
 List<String> grinderArgs() => _args;
 List<String> _args;
+bool _verifyProjectRoot;
 
-Future handleArgs(List<String> args, {bool verifyProjectRoot: true}) {
+Future handleArgs(List<String> args, {bool verifyProjectRoot}) {
   _args = args == null ? [] : args;
+  _verifyProjectRoot = verifyProjectRoot;
+  return script.execute(grinderArgs());
+}
 
-  ArgParser parser = createArgsParser();
-  ArgResults results = parser.parse(grinderArgs());
-
-  if (results['version']) {
+@Command(help: 'Dart workflows, automated.', plugins: const [const Completion()])
```

Comment:
This is the line that's causing the analyzer to crash (see http://dartbug.com/23354).  If you don't want to wait for the bug fix, you should be able to work around the problem with no change in semantics by moving the instance creation expression to a separate constant, e.g.:

    const _completion = const Completion();
    @Command(help: 'Dart workflows, automated.', plugins: const [_completion])